tests/drivers/spi/spi_loopback/overlay-stm32-spi-16bits-dma-no-nocache.conf

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2023 Graphcore Ltd, All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# enable DMA mode for SPI loopback test
+CONFIG_SPI_STM32_DMA=y
+# CONFIG_SPI_STM32_INTERRUPT is not set
+# CONFIG_SPI_ASYNC is not set
```

Comment:
Please use :
```
CONFIG_SPI_STM32_INTERRUPT=n
CONFIG_SPI_ASYNC=n
```
This will be consistent with other .conf files and will take care of cases where symbols are defined in board config for instance.

Reply:
Done.